lighthouse-plots/analyze.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const utils = require('./utils');
+
+const OUT_PATH = path.resolve(__dirname, 'out');
+const GENERATED_RESULTS_PATH = path.resolve(OUT_PATH, 'generatedResults.js');
+const LIGHTHOUSE_RESULTS_FILENAME = 'lighthouse.json';
+
+const METRICS = [
```

Comment:
this might be addressed anyhow, but I didn't see why this was neccessary. Isn't it the same as `object.keys` on the `readResult` value?